python/tk_framework_desktopserver/shotgun/api_v2.py

```diff
@@ -305,6 +331,54 @@ def get_actions(self, data):
             data,
         )
 
+        # The first thing we do is check to see if we're dealing with a
+        # classic SGTK setup. In that case, we're going to short-circuit
+        # the get_actions call and go into a legacy setup that makes use
+        # of the "tank" command by way of this api's process_manager.
+        config_entities = [d["entity"] for i, d in all_pc_data.iteritems()]
+        legacy_config_data = dict()
+
+        # A classic-style config will contain a roots.yml file that defines
+        # a set of storage roots. We can access that by way of the descriptor
+        # object that we already have. If there are required storages, then
+        # we know we're in a classic situation.
+        for config in config_entities:
+            if config["descriptor"].required_storages:
+                legacy_config_data[config["name"]] = (
+                    config["descriptor"].get_path(),
```

Comment:
If [this comment](https://github.com/shotgunsoftware/tk-core/pull/469#pullrequestreview-42064764) is correct, it means that this line is not valid anymore as well as any other places where you call `get_path` either.